### PR TITLE
Fix websockets

### PR DESCRIPTION
**Pull request by rozetko**
This PR fixes the case when connection to server in analytics drops:
https://github.com/hastic/hastic-server/pull/814#issuecomment-588322888
https://github.com/hastic/hastic-server/pull/814#issuecomment-594709568

// TODO: more description

### CODE REVIEW

File: analytics/analytics/services/server_service.py [hunk 1]
```diff
@@ -94,19 +97,32 @@ def __aiter__(self):
                 asyncio.ensure_future(self.__handle_ping())
             else:
                 asyncio.ensure_future(self._send_message_from_thread(received_string))
+
+    async def __reconnect(self):
+        if not self.__reconnecting:
+            self.__reconnecting = True
```
Comment (jonyrock-back) [inline analytics/analytics/services/server_service.py]: try to make a task which exists only in one instanse and try to await it here if it exist...or make a new task (which will be the only task)

**Comment by jonyrock** [comment]
It's very important to explain what you found interesting during your research

**Comment by jonyrock** [comment]
@rozetko please finish this work in two days

**Comment by rozetko** [comment]
Moved to https://github.com/hastic/analytics/pull/17

**Pull request closed without merging**